components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImpl.java

```diff
@@ -126,6 +127,30 @@ public boolean validateRequestObject(RequestObject requestObject, OAuth2Paramete
                 return false;
             }
         }
+
+        try {
+            if (OAuth2Util.isFapiConformantApp(requestObject.getClaimValue(Constants.CLIENT_ID))) {
```

Comment:
Move to a private method

Reply:
Move to the top

Reply:
fixed with bc5244f236cdc227908ca0dcb2ecb98ea38fafcb